src/content/docs/new-relic-control/agent-control/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Agent Control overview"
+metaDescription: "Agent Control is a lightweight agent supervisor that manages other New Relic and New Relic distribution of OpenTelemetry agents by communicating with Fleet Control to remotely configure, update, and monitor the health of infrastructure-level instrumentation."
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+Agent Control is a lightweight agent supervisor that manages other New Relic and New Relic distribution of OpenTelemetry agents. It communicates with Fleet Control to remotely configure, update, and monitor the health of infrastructure-level instrumentation on Kubernetes.
+## Prerequisites
+Before you begin deploying Agent Control to your Kubernetes clusters, ensure that you meet the following prerequisites:
+
+- **Helm 3:** Helm version 3 must be installed on your machine. For installation instructions, see [Installing Helm](https://helm.sh/docs/intro/install).
+- **New Relic license key:** You'll need a license key to report telemetry to your New Relic account.
+- **Kubernetes cluster name:** Have the name of your Kubernetes cluster ready. You'll reference it during the installation process.
+- **Auth Domain Manager:** To set up Agent Control and connect it to Fleet Control, you must have the Authentication Domain Manager role. For more info, refer to [User Management Concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#admin-settings).
+- **Existing instrumentation:** If your Kubernetes cluster is already instrumented with New Relic, you must uninstall the existing instrumentation before installing Agent Control. After installing Agent Control, you can use Fleet Control to install and manage infrastructure agents and instrumentation on the cluster. For guidance, refer to [Manage existing instrumentation with Agent Control](/docs/new-relic-control/agent-control/reinstrumentation).
+
+## Compatibility
+
+Agent Control is compatible with:
+- Last three minor Kubernetes releases
```

Comment:
Agent Control will not even install if the k8s version is not matching this requirement because of a check the flux chart does, should we be more explicit?  Should we state which specific version is the minimum required? (1.30)